src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. You can consider adding some validation for the input parameters in the public methods to ensure that unexpected inputs do not cause issues.